t/14-grutasks.t

```diff
@@ -644,10 +645,12 @@ subtest 'handling dying GRU task' => sub {
     like $associated_job->reason, qr/^preparation failed: Thrown fail at/, 'reason of associated job set';
 };
 
+
+
```

Comment:
please review your changes for such whitespace noise before submitting.

Reply:
Keep in mind that this is a draft